packages/wonder-blocks-popover/components/popover-content.test.js

```diff
@@ -32,34 +34,29 @@ describe("PopoverContent", () => {
         expect(onCloseMock).toBeCalled();
     });
 
-    it("should warn when setting PopoverContext.Provider and onClose together", () => {
-        expectRenderError(
-            <PopoverContext.Provider value={{close: () => {}}}>
-                <PopoverContent
-                    title="title"
-                    content="content"
-                    onClose={() => {}}
-                />
-                ,
-            </PopoverContext.Provider>,
-            "You've specified 'onClose' on the content when using Popover. Please specify 'onClose' on the Popover instead",
-        );
-    });
-
     it("should warn when setting a image and icon at the same time", () => {
-        expectRenderError(
+        // Arrange
+        const nodes = (
             <PopoverContent
                 title="illustration"
                 content="content"
-                image={<img src="/dummy-image.png" alt="dummy" />}
-                icon="icon.png"
-            />,
+                image={<img src="/dummy-image.png" alt="popover image" />}
+                icon={<img src="/dummy-icon.png" alt="popover icon" />}
+            />
+        );
+
+        // Act
+        const underTest = () => mount(nodes);
+
+        // Assert
+        expect(underTest).toThrowError(
```

Comment:
Why not toThrowErrorMatchingInlineSnapshot? That way it will auto update for us too.

Reply:
👍 TIL. This matcher is awesome! thanks for the suggestion :)